### PR TITLE
 After a post or "like" only the single thread is now updated

### PR DESCRIPTION
**Pull request by annando**
Liking or commenting now only updates the single thread. Before this change it could have happened that after this action the background process fetched much more posts than just this single thread.

Additionally the whole JS script ```main.js``` had been reformatted since it looked really ugly.

### CODE REVIEW

**Comment by MrPetovan** [comment]
Yes! At long last!

**Comment by annando** [comment]
I just added a change so that only toplevel posts are fetched during an update when ordered after posts. This should improve the experience.

File: mod/update_network.php [hunk 1]
```diff
@@ -12,14 +12,15 @@
 
 function update_network_content(App $a)
 {
-	$profile_uid = intval($_GET["p"]);
+	$profile_uid = intval($_GET['p']);
+	$parent = intval($_GET['item']);
```
Comment (MrPetovan) [inline mod/update_network.php]: What if the GET parameter isn't set?

Comment (annando) [reply mod/update_network.php]: It is always set: https://github.com/friendica/friendica/pull/4514/files#diff-a3a4de556a72540dcdb30e5cacd17de3R422

**Pull request merged**